CONTRIBUTING.md

```diff
@@ -21,6 +21,7 @@ The following guide details the steps to setup a local development environment f
 
 ### Prerequisites
 
+- [GCC](https://gcc.gnu.org/)
```

Comment:
only on Linux, no?. It is needed for golang to do dynamic linking (maybe note that as well?)

Reply:
good point, thanks